_posts/2024-12-02-ranger.md

```diff
@@ -0,0 +1,94 @@
+in 2022, followed quickly by the addition of the Trino support to the Apache
+Ranger repository.
+
+## Testing and container images
+
+However that was only half of the needed support. The Trino project moves very
+fast with nearly weekly releases, so the best approach is to have the supporting
+plugin in Trino directly so every release includes the relevant updates. [Erik
+Anderson](https://github.com/dprophet) created a more mature plugin that was in
+production use for quite a while for Trino. His [pull request from July
+2022](https://github.com/trinodb/trino/pull/13297) included great background
+reasoning for having the plugin in Trino. One of the issues that Erik solved for
+the Trino project is testing. Trino plugins require the availability of a
+container image for testing whatever integration. Apache Ranger did still not
+ship a container in 2022, but thanks to the lobbying efforts of Erik this
+changed and a container image became available over the months.
+
+## A long sprint
+
+Unfortunately focus changed and while the PR from Erik existed and was useful,
+it never made it to merge due to waning priorities. That changed when [Madhan
+Neethiraj](https://github.com/mneethiraj) from the Apache Ranger project stepped
+up and created [new PR](https://github.com/trinodb/trino/pull/22675) in July 2024. 
+
+We knew this could be another shot at it and it would require a lot of work to
+get it done, since we put a high focus on quality so that we can maintain the
+Trino codebase for the long run. Monitoring all PRs regularly [Manfred
+Moser](https://github.com/mosabua) noticed it and jumped in with first help.
+
+Erik and other interested users chimed in.
+[lozbrown](https://github.com/lozbrown) and Manfred helped with documentation
+and getting other developers interested. The heavy technical reviews and lots of
+guidance came from [Krzysztof Sobolewski](https://github.com/ksobolew) and
+[Grzegorz Kokosiński](https://github.com/kokosing).
+
+During the whole process Madhan had to react to comments, update the code, and
+also regularly rebase his PR to adjust for the constantly changing Trino
+codebase in the master branch. Starburst recognized Madhan's effort and
+[featured him as Starburst Trino
+Champion](https://www.starburst.io/community/trino-champions/). Interestingly,
+the container image ended up not being used for testing, however it will be
+crucially important for many users deploying Apache Ranger on Kubernetes anyway.
+Nearly 400 comments and over four months later we all got to celebrate. The
+Trino maintainer Grzegorz took on the responsibility and merged the PR. [Yuya
+Ebihara](https://github.com/ebyhr) and [Martin
+Traverso](https://github.com/martint) followed up with
+[minor](https://github.com/trinodb/trino/pull/24238)
+[cleanups](https://github.com/trinodb/trino/pull/24252), and we finally shipped
+the plugin as part of [Trino
+466]({{site.baseurl}}/docs/current/release/release-466.html).
```

Comment:
I think a section on how to get started or a link to the docs on what's going on would be good. I finish this blog knowing the amount of work that went into getting it merged, but not how to use it or why I would want to.

Reply:
Love this, he's right.

Reply:
Added a little paragraph after the congratulations now .. and also in the opening